anda/system/surface-dtx-daemon/surface-dtx-daemon.spec

```diff
@@ -1,13 +1,14 @@
 %global debug_package %{nil}
 %global ver v0.3.8-1
+%global ver2 %(echo %{ver} | sed 's/^v//')
 
 Name:           surface-dtx-daemon
 Version:        %(echo %ver | sed 's/-/~/g')
 Release:        1%{?dist}
 Summary:        Surface Detachment System (DTX) Daemon
 License:        MIT
 URL:            https://github.com/linux-surface/surface-dtx-daemon
-Source:         %url/archive/refs/tags/v%ver.tar.gz
+Source:         %url/archive/refs/tags/%ver.tar.gz
```

Comment:
if the tag does not start with `v`, then the auto update script would not add a `v`. This contradicts with the definition `%global ver v0.3.8-1`.

Reply:
if there is a new version / tag to `linux-surface/surface-dtx-daemon`, please bump `ver` manually.

also, you might no longer need `%ver2` depending on the situation

Reply:
Well yeah, the surface-dtx-daemon releases do have the v in them, so the url it was trying to download had 2 'v's in it.
If I really need the v before %ver in the source url, manual bumping is fine, but I'd rather not?